basecamp_client.py

```
@@ -1365,3 +1366,204 @@ def get_upload(self, project_id, upload_id):
+        byte_size = meta.get("byte_size")
+        if (
+            max_bytes is not None
+            and byte_size is not None
+            and byte_size > max_bytes
+        ):
+            raise Exception(
+                f"Upload size {byte_size} bytes exceeds max_bytes={max_bytes}. "
+                f"Increase max_bytes or fetch the file via the Basecamp UI."
+            )
+
+        response = requests.get(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)